import sys

print('Internal error in script', file=sys.stderr)
exit(-1)
